src/components/Footer/Footer.tsx

```diff
@@ -17,12 +20,28 @@ const Footer = () => {
     <Wrapper>
       <LogoImage id="logo-oca-white" width={200} height={150} />
       <References>
-        {Object.entries(sections).map(([key, item]) => (
-          <Sections key={key} href={item.path || ""}>
-            {item.name}
-          </Sections>
-        ))}
-        <Divider />
+        <Sections>
+          {Object.entries(sections).map(([key, item]) => (
+            <InnerSections key={key}>
+              <SectionTitle>
+                {item.name}
+                <Divider />
+              </SectionTitle>
+              {item.children != undefined ? (
```

Comment:
```suggestion
              {item.children ? (
```

We don't need the != (actually in JS would be !==). With `item.children` alone we can make the condition work as well